products/vmware_cb_enterprise_edr.py

```diff
@@ -38,18 +38,27 @@ def _convert_relative_time(relative_time) -> str:
 
 class CbEnterpriseEdr(Product):
     product: str = 'cbc'
+    profile: str = "default"
```

Comment:
⛏️ Can more of the internal class variables (e.g. `url`, `token`, `profile`, etc.) be defined up here with their types?

```suggestion
    profile: str = "default"
    url: Optional[str] = None
    token: Optional[str] = None
    org_key: Optional[str] = None
    _device_group: Optional[list[str]] = None
    _device_policy: Optional[list[str]] = None    
```